app/models/tag.rb

```diff
@@ -89,6 +89,10 @@ def validate_name
     errors.add(:name, "contains non-ASCII characters") unless name.match?(/\A[[a-z0-9]]+\z/i)
   end
 
+  def errors_as_sentence
+    errors.full_messages.to_sentence
+  end
```

Comment:
This needs to be added here since the `Tag` model doesn't inherit from `ApplicationRecord`.